Source/SvgElementStyle.cs

```diff
@@ -469,36 +466,23 @@ internal IFontDefn GetFont(ISvgRenderer renderer)
                 return new SvgFontDefn(font, fontSize, OwnerDocument.Ppi);
             }
         }
-#if !NETSTANDARD20
-        public static System.Drawing.Text.PrivateFontCollection PrivateFonts = new System.Drawing.Text.PrivateFontCollection();
-#endif
-        public static object ValidateFontFamily(string fontFamilyList, SvgDocument doc)
+
+        public static object ValidateFontFamily(string fontFamilyList, SvgDocument doc, SvgFontManager fontManager)
         {
             // Split font family list on "," and then trim start and end spaces and quotes.
             var fontParts = (fontFamilyList ?? string.Empty).Split(new[] { ',' }).Select(fontName => fontName.Trim(new[] { '"', ' ', '\'' }));
-            FontFamily family;
-            IEnumerable<SvgFontFace> sFaces;
 
             // Find a the first font that exists in the list of installed font families.
-            //styles from IE get sent through as lowercase.
+            // styles from IE get sent through as lowercase.
```

Comment:
What about that IE-specific part (lower-case font names)? Isn't it needed anymore?